### PR TITLE
Delete endpoint

### PR DESCRIPTION
**Pull request by Ainkharan**
Delete by Listing ID, Member ID With test

### CODE REVIEW

File: src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java [hunk 1]
```diff
@@ -90,5 +90,42 @@ void testGetListingByIdWhenListingDoesNotExistThenReturnNotFound() throws Except
...
+    // Delete Testings
+    @Test
+    void testDeleteListing_Success() throws Exception {
+        Long listingID = 1L;
+
+        mockMvc.perform(MockMvcRequestBuilders.delete("/api/v1/listings/{listingID}", listingID))
+                .andExpect(MockMvcResultMatchers.status().isNoContent());
+
+        verify(mockListingService).deleteListingById(listingID);
+    }
```
Comment (Lonrot) [inline src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java:102]: use times from Mockito to see that it is being called only once.
Change the expected code to accepted( we accepted the request to delete and was successful)

File: src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java [hunk 2]
```diff
@@ -90,5 +90,42 @@ void testGetListingByIdWhenListingDoesNotExistThenReturnNotFound() throws Except
...
+
+    @Test
+    void testDeleteListingsByMember_NotFound() throws Exception {
+        Long memberID = 1L;
+        doThrow(new ListingNotFoundException("No listings found for member")).when(mockListingService).deleteListingsByMember(memberID);
+
+        mockMvc.perform(MockMvcRequestBuilders.delete("/api/v1/listings/member/{memberID}", memberID))
+                .andExpect(MockMvcResultMatchers.status().isNotFound());
+    }
```
Comment (Lonrot) [inline src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java:130]: all good. I would start changing the expected HTTP codes and refactor.

**Review by Lonrot — CHANGES_REQUESTED** [review]
Thank you for your work Nada, just a few things to change. Also, correct me if I am mistaken but can't find any service test. Did you forget to commit?

File: src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java (hunk 2 above)
Comment (Ainkharan) [reply src/test/java/springsamurais/toyswapbackend/controller/ListingControllerTest.java:143]: HTTP codes Changed and refactored

**Review by Lonrot — COMMENTED** [review]
We need to change the controller annotations, no merge yet. Might change it later myself and merge it.

**Pull request merged**